src/providers/snippetCompletionProvider.ts

```diff
@@ -7,14 +7,15 @@ export class SnippetCompletionProvider implements vsc.CompletionItemProvider {
 	constructor(private config: Configuration) {
 	}
 
-	provideCompletionItems(document: vsc.TextDocument, position: vsc.Position, _token: vsc.CancellationToken, _context: vsc.CompletionContext) {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	provideCompletionItems(document: vsc.TextDocument, position: vsc.Position, _token: vsc.CancellationToken, _context: vsc.CompletionContext): vsc.CompletionItem[] {
 		const text = Document.getTextBeforeCursor(document, position);
 
-		if (this.config.snippets && SnippetTriggerRegex.test(text)) {
-			return Object.keys(this.config.snippets).map(name => {
+		if (this.config && this.config.snippets && SnippetTriggerRegex.test(text)) {
```

Comment:
I think optional chaining operator could be used here since you have already updated Typescript so that would become `this.config?.snippets`